libraries/go/riot-api-golang.json

```
@@ -0,0 +1,19 @@
+{
+  "owner": "junioryono",
+  "repo": "Riot-API-Golang",
+  "language": "Go",
+  "links": [
+    {
+      "name": "github",
+      "url": "https://github.com/junioryono/Riot-API-Golang"
+    }
+  ],
+  "metadata": {
+    "version": "0.1"
+  },
+  "tags": [
+    "v5",
```

you forgot to add the `v4` tag
```suggestion
    "v4",
    "v5",
```